index.html

```diff
@@ -145,7 +146,7 @@ <h3 class="h4 mb-2 text-light">Sneaky Live</h3>
         <div class="col-lg-8 text-center">
           <h2 class="mt-0 text-uppercase">Contacto</h2>
           <hr class="divider my-4">
-          <p class="text-muted mb-5">Si estás interesado en unirte, tenés alguna sugerencia o te gustaría contribuir con nosotros no dudes en escribirnos!</p>
+          <p class="text-muted mb-5">If you are interested on getting involved with our team, make a suggestion or want to contribute with us, just write us!</p>
```

Comment:
"If you are interested in getting involved, making a suggestion or if you want to contribute with us, shoot us an email!"